### PR TITLE
[do not merge/pending obs table] obsolete/merge Okamoto syndrome

### LINKED ISSUE #6335: MONDO:0011499 Okamoto & MONDO:0014700 Au-Klein[Merge]
**Mondo term (ID and Label)**
MONDO:0011499 Okamoto syndrome

**Reason for deprecation**
OMIM merged: 604916 MOVED TO [616580](https://www.omim.org/entry/616580)

**Term to be merged with**
MONDO:0014700 Neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation  (aka Au-Klein syndrome, which is OMIM's pref name)

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6335

merge
MONDO:0011499 Okamoto syndrome

with
MONDO:0014700 Neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation (aka Au-Klein syndrome, which is OMIM's pref name)

@twhetzel I assigned you to review this PR, and I made you a video on ['Review Merge PR'](https://drive.google.com/file/d/1kE_C2BB0e1yfzoFc5rwmeleEredEOlvV/view?usp=drive_link) in this [folder](https://drive.google.com/drive/u/0/folders/1cyE7zRxKcj7ccclFqAyDW2mtmx0iHlBm)

Note, there are a couple other videos in that folder that may be helpful to watch:
- [Merge terms in Mondo Part 1](https://drive.google.com/file/d/1Njo_cSwXh28rJpqMaz729aNRUwWVpzZw/view?usp=drive_link)
- [Merge terms in Mondo Part 2](https://drive.google.com/file/d/1abkIZsbLTGfQQcrbHYlXUoGalbWQOul9/view?usp=drive_link)

### CODE REVIEW

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293916]: Based on https://www.omim.org/entry/616580 should the name be `Au-Kline syndrome` vs. `Klein`?

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293945]: Why was this transferred as a `seeAlso` vs. a `subset` as found in the merge obsolete term 
(`subset: gard_rare {source="GARD:0004064"}`)? The subset `ordo_malformation_syndrome` was transferred as a `subset`.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293922]: Why is this marked as `RELATED DEPRECATED`? I ask because it is shown here: https://rarediseases.org/gard-rare-disease/okamoto-syndrome/

Also, why are there 2 synonym axioms with `[GARD:0004064]` for both by one is `RELATED` (two lines above) and this one is `RELATED DEPRECATED`?

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293924]: Why and how (manual change?) was this changed from `RELATED DEPRECATED` to `EXACT DEPRECATED`?

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293931]: How does this line relate to Line 293929 since the Orphanet and OMIM identifiers are the same but only the `xref: OMIM:604916` has `source="MONDO:equivalentObsolete`?

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293929]: I don't follow why this is `MONDO:equivalentObsolete` since the term it is applied to after the merge is not the obsolete term.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293934]: What happened to `source="MONDO:equivalentTo` from the obsoleted term?

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:293936]: why is this `source` now `source="MONDO:equivalentObsolete"`>

**Review by twhetzel — COMMENTED** [review]
Comments/questions inline.

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293916]: good catch! I will fix this

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293922]: here is some info about deprecated synonyms: https://mondo.readthedocs.io/en/latest/editors-guide/changelog-metadata-schema/#glossary

and here:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293924]: yes, I changed this manually. All of the OMIM synonyms should generally be exact. We have a ticket to update this:
https://github.com/monarch-initiative/mondo/issues/2629

**Comment by nicolevasilevsky** [comment]
@twhetzel I am still reviewing your comments, I'll try to finish this tomorrow

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293931]: this OMIM class was merged into [OMIM:616580](https://omim.org/entry/616580), hence we merged it in Mondo. But yes, Orphanet does have this class as equiv to the OMIM class, so Orphanet should probably merge these terms too. They only do releases once a year, I believe, so we probably won't see this merge in Orphanet until their release.

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293934]: this would be a [proxy merge](https://mondo.readthedocs.io/en/latest/editors-guide/proxy-merge/), so I removed one equivalent axiom here

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:293945]: we don't have this documented in the documentation, maybe we should add this somewhere.

we used to add seeAlso for any URL (including the github URLs) but recently we started using the term tracker item annotation instead of seeAlso, but we still use seeAlso for other URLs, like GARD.

**Comment by nicolevasilevsky** [comment]
@twhetzel I finished commenting on your comments. Let me know if you have further questions. I'm traveling all week but I may be able to jump on a call later in the week if needed.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:293945]: yeah, the difference in how the two are handled is what caught my eye and is why I asked. 

NOTE: there is also `http://www.w3.org/2000/01/rdf-schema#seeAlso` that a number of ontologies in OLS use as well as `http://www.geneontology.org/formats/oboInOwl#seeAlso` that a smaller number of ontologies use that could be good to use instead of free text `seeAlso` and hopefully would not break anything downstream.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:293922]: Thanks, I see the reason now and also realize that `RELATED DEPRECATED` is actually two tags.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:293929]: Ah, so the source as `MONDO:equivalentObsolete` is not a result of obsolete/merge action.

**Review by twhetzel — APPROVED** [review]
Thanks for updating the term name and answering all my questions!

Since @sabrinatoro asked that further obsolete/merge PRs not be merged into master due to the work on mass obsoletion, can this be reverted back to a Draft PR or re-named to Not Ready to Merge?

**Comment by nicolevasilevsky** [comment]
I'll rename it to do not merge

**Pull request merged**
